### ISSUE TITLE
Liquid Crystal display issue

### ISSUE THREAD

**Issue by FateForWindows** [Wolvan/Homebr3w#34]
Pokemon Liquid Crystal has a corrupt icon and blank information listed in Homebr3w. Additionally, somehow Homebr3w thinks Shovel Knight Update 2.2 is homebrew! It also has a different icon (noise) and says something about the SMDH being messed up (likely because it's not homebrew).

**Comment by Wolvan** [comment]
@ksanislo can you check the what shares the Shovel Knight Update TID, plesae?

**Comment by ksanislo** [comment]
@FateForWindows Can you check if this is still a problem? I've removed the entry that I suspect was the conflict, but I don't have an easy way to find the TID of the shovel knight update.

**Comment by FateForWindows** [comment]
Nope, wasn't it. Tried installing to see what it was and it killed my save but at least I know it's ~88 MB in size. I'll reinstall the update and check the titleid with FBI.

**Comment by FateForWindows** [comment]
Nope, wasn't it. Tried installing to see what it was and it killed my save but at least I know it's ~88 MB in size. I'll reinstall the update and check the titleid with FBI.

**Comment by FateForWindows** [comment]
Titleid is 0004000E00119A00
